### PR TITLE
Release notes for 1.14.5

### PR DESCRIPTION
**Pull request by antontroshin**

### CODE REVIEW

File: docs/release_notes/v1.14.5.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+### Problem
+
+Transactions were not respecting the metadata in the transaction request.
+For example, content type specified in the metadata was not being applied and the message was being sent as plain text.
+
+### Impact
+
+Outbox transactions were not respecting the metadata. 
+Cloudevents data was being sent as plain text and Cloudevents Middleware was not able to parse the data resulting in 415 error code.
```
Comment (JoshVanL) [inline docs/release_notes/v1.14.5.md:17]:
```suggestion
Outbox transactions were not respecting metadata. 
Cloudevents Middleware was unable to parse the metadata, resulting in 415 error codes.
```

File: docs/release_notes/v1.14.5.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Dapr 1.14.5
+
+This update includes bug fixes:
+
+- [Fix content type metadata from query for outbox transaction](#fix-content-type-metadata-from-query-for-outbox-transaction)
```
Comment (JoshVanL) [inline docs/release_notes/v1.14.5.md:5]:
```suggestion
- [Fix content type metadata from query for outbox transactions](#fix-content-type-metadata-from-query-for-outbox-transaction)
```

File: docs/release_notes/v1.14.5.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ 
+## Fix content type metadata from query for outbox transaction
```
Comment (JoshVanL) [inline docs/release_notes/v1.14.5.md:7]:
```suggestion
## Fix content type metadata from query for outbox transactions
```

File: docs/release_notes/v1.14.5.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+### Problem
+
+Transactions were not respecting the metadata in the transaction request.
```
Comment (JoshVanL) [inline docs/release_notes/v1.14.5.md:11]:
```suggestion
Outbox transactions were not respecting the metadata in the transaction request.
```

**Review by JoshVanL — CHANGES_REQUESTED** [review]

File: docs/release_notes/v1.14.5.md (hunk 2 above)
Comment (antontroshin) [reply docs/release_notes/v1.14.5.md]: Done

File: docs/release_notes/v1.14.5.md (hunk 3 above)
Comment (antontroshin) [reply docs/release_notes/v1.14.5.md]: Done

File: docs/release_notes/v1.14.5.md (hunk 4 above)
Comment (antontroshin) [reply docs/release_notes/v1.14.5.md]: Done

File: docs/release_notes/v1.14.5.md (hunk 1 above)
Comment (antontroshin) [reply docs/release_notes/v1.14.5.md]: Done

**Review by JoshVanL — APPROVED** [review]

**Pull request merged**